app/connectors/MarriageAllowanceConnector.scala

```diff
@@ -69,7 +69,9 @@ class MarriageAllowanceConnector @Inject()(httpClient: HttpClient, applicationCo
         case response if response.status == 200 => Right(response.json.as[GetRelationshipResponse])
         case errorResponse => Left(errorResponse.json.as[MarriageAllowanceError])
       }.recoverWith {
-      case t: Throwable => Future.failed(t)
+          case t: Throwable => {
```

Comment:
In Scala when we have a function expression like this case match; if the change is on line of code we don't need to add braces round the code. See line 70

Reply:
Removed braces